subtrack.DAL/Entities/Subscription.cs

```diff
@@ -26,6 +26,8 @@ public class Subscription : ICloneable
     [Range(1, int.MaxValue, ErrorMessage = "Interval has to be greater than 0")]
     public int BillingInterval { get; set; }
 
+    public string BackgroundColor { get; set; }
```

Comment:
On the home page and monthly, display the color as the background color for the card - style background-color:@Sub.PrimaryColor

Reply:
Done